### PR TITLE
Feature/cyberint feed

### PR DESCRIPTION
**Pull request by TalGumi**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Cyberint/Integrations/Cyberint/Cyberint.yml [hunk 1]
```diff
@@ -13,6 +13,11 @@ configuration:
   name: environment
   required: true
   type: 0
+- additionalinfo: The customer name.
+  display: Customer name
+  name: customer_name
+  type: 0
+  required: false
```
Comment (jbabazadeh) [inline Packs/Cyberint/Integrations/Cyberint/Cyberint.yml:20]: Please add the customer_name parameter to the README file.

File: Packs/Cyberint/Integrations/FeedCyberint/README.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+
+## Configure Cyberint Feed on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Cyberint Feed.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | The Cyberint IOC feed endpoint URL |  | True |
```
Comment (jbabazadeh) [inline Packs/Cyberint/Integrations/FeedCyberint/README.md:12]: Same here please add customer name.

File: Packs/Cyberint/Integrations/FeedCyberint/README.md [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+
+#### Base Command
+
+`cyberint-get-indicators`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| limit | The maximum number of results to return. The default value is 10. Default is 10. | Optional |
```
Comment (jbabazadeh) [inline Packs/Cyberint/Integrations/FeedCyberint/README.md:51]:
```suggestion
| limit | The maximum number of results to return.  Default is 10. | Optional |
```

File: Packs/Cyberint/Integrations/FeedCyberint/README.md [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+| Cyberint.type | String | The indicator type. |
+| Cyberint.Tags | String | Tags that are associated with the indicator. |
+| Cyberint.description | String | The feed description. |
+| Cyberint.detected_activity | String | The feed detected activity. |
+| Cyberint.observation_date | String | The feed observation date. |
+| Cyberint.severity_score | String | The feed severity score. |
+| Cyberint.confidence | String | The feed confidence. |
+
+#### Command example
+```!cyberint-get-indicators limit=10 execution-timeout=700```
```
Comment (jbabazadeh) [inline Packs/Cyberint/Integrations/FeedCyberint/README.md:67]:
```suggestion
```!cyberint-get-indicators limit=10```
```

File: Packs/Cyberint/ReleaseNotes/2_0_0.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+
+#### Integrations
+
+##### New: Cyberint Feed
+
+- New: Use the Cyberint Feed integration to get indicators from the feed. (Available from Cortex XSOAR 6.9.0).
+
+##### Cyberint
+
+- Added new *customer_name* parameter to the instance settings.
```
Comment (jbabazadeh) [inline Packs/Cyberint/ReleaseNotes/2_0_0.md:10]:
```suggestion
- Added the **Customer name** parameter to the instance settings.
```

**Review by jbabazadeh — CHANGES_REQUESTED** [review]
Hi @TalGumi,
Thank you for your contribution!
Great work!! 😃 
I have a few minor comments.

Please feel free to reach out to me with any questions - I'm available here or on slack

File: Packs/Cyberint/Integrations/FeedCyberint/README.md (hunk 4 above)
Comment (TalGumi) [reply Packs/Cyberint/Integrations/FeedCyberint/README.md:67]: The command will return a time-out error when not using the `execution-timeout` argument.

**Review by jbabazadeh — COMMENTED** [review]
Hi @TalGumi  the code looks good!

We're ready for a demo. Please check [this page](https://xsoar.pan.dev/docs/contributing/demo-prep), and let me know when you're available for one over DFIR.

**Comment by jbabazadeh** [comment]
Hi @TalGumi, as discussed over Slack, we are waiting for the partner to make changes to the API so you can implement the feed in the best way. I'll close this PR for now.

**Pull request closed without merging**